### PR TITLE
[15.0][ADD] sale_fully_delivered

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Split from https://github.com/OCA/sale-workflow/pull/2306

According to this comment: https://github.com/OCA/sale-workflow/pull/2306#pullrequestreview-1218920523

This module introduce a new field that indicates
the Sales Order is fully delivered. It is useful to know if the SO is closed

cc @ForgeFlow

### CODE REVIEW

File: sale_fully_delivered/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2022 ForgeFlow S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+{
+    "name": "Sales Fully Delivered and Invoiced",
+    "summary": "Useful filters in Sales to know the actual status of deliveries "
```
Comment (MiquelRForgeFlow) [inline sale_fully_delivered/__manifest__.py]:
```suggestion
    "summary": "Useful filters in Sales to know the actual status of deliveries."
```

**Review by JuanyDForgeflow — APPROVED** [review]
Technical LGTM

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

File: sale_fully_delivered/models/sale.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+    is_fully_delivered = fields.Boolean(
+        compute="_compute_is_fully_delivered", store=True, index=True
+    )
+
+    @api.depends("order_line.qty_delivered")
+    def _compute_is_fully_delivered(self):
+        for sale in self:
+            if all(
+                line.product_uom_qty <= line.qty_delivered
```
Comment (rousseldenis) [inline sale_fully_delivered/models/sale.py]: @AaronHForgeFlow float_compare ?

File: sale_fully_delivered/tests/test_sale_fully_delivered_invoiced.py [hunk 3]
```diff
@@ -0,0 +1,91 @@
+# Copyright 2022 ForgeFlow, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo.tests import common, tagged
+
+
+@tagged("-at_install", "post_install")
+class TestSaleFullyDeliveredInvoiced(common.TransactionCase):
+    def setUp(self):
```
Comment (rousseldenis) [inline sale_fully_delivered/tests/test_sale_fully_delivered_invoiced.py]: Change this to class method

**Comment by MiquelRForgeFlow** [comment]
@rousseldenis comments attended

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by rousseldenis** [comment]
@MiquelRForgeFlow @AaronHForgeFlow In fact, testing functionally, I've found this module : https://github.com/OCA/sale-workflow/tree/15.0/sale_delivery_state

So, this one is duplicate.

**Comment by AaronHForgeFlow** [comment]
Yes, I think I can use that one. Thanks.

**Pull request closed without merging**